x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_endpoint/render_cohere.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React from 'react';
+import { RenderEndpointProps } from './render_endpoint';
+import { ModelBadge } from './model_badge';
+
+interface ServiceSettings {
+  embedding_type: string;
+  model_id: string;
+}
+
+interface TaskSettings {
+  input_type: string;
+  truncate: string;
+}
+
+export const RenderCohere: React.FC<RenderEndpointProps> = ({ endpoint }) => {
+  const serviceSettings = endpoint.service_settings as ServiceSettings;
```

Comment:
Please don't type cast. With rare exceptions, if you're type casting you messed up earlier in your type construction, or you need to explicitly check what type this variable is. This is overriding the type system, which is going to lead to bugs down the line.

Reply:
Updated.